saturday-afternoon/components/FlightItem.js

```diff
@@ -1,27 +1,38 @@
 // @flow
 
 import * as React from "react";
-import { Row, Timeline, Button } from "antd";
+import { Row, Button } from "antd";
+
+import Legs from "./Legs";
+import resolveScopedStyles from "../utils/resolveScopedStyles";
 
 type Flight = {};
 
 type Props = {
   flight: Flight
 };
 
+const buttonStyles = resolveScopedStyles(
+  <scope>
+    <style jsx>{`
+      .buy {
+        width: 100%;
+      }
+    `}</style>
+  </scope>
+);
+
 const FlightItem = (props: Props) => (
   <div>
     <Row>
-      <Timeline>
-        <Timeline.Item>step1</Timeline.Item>
-        <Timeline.Item>step2</Timeline.Item>
-      </Timeline>
+      <Legs />
     </Row>
     <Row>
-      <Button type="primary" style={{ width: "100%" }}>
+      <Button type="primary" className={`buy ${buttonStyles.className}`}>
```

Comment:
again, I do not like usage of resolveScopedStyles and className. did you come up with this approach yourself or get somwhere?